packages/unmock-core/src/runner/index.ts

```diff
@@ -9,16 +9,15 @@ const defaultRunnerOptions: IRunnerOptions = {
 };
 
 const errorHandler = (
-  allFailed: boolean,
+  nTimes: number,
   errors: Error[],
   intermediaryErrors: Array<string | { message: string }>,
   cb?: jest.DoneCallback,
 ) => {
   // tslint:disable-next-line:max-line-length
   const msg = (rest: string) =>
-    `${chalk.red("@unmock")} - ${
-      allFailed ? "all" : "some"
-    } tests failed. Here's the first error.\n${rest}`;
+    `This many tests failed: ${errors.length +
+      intermediaryErrors.length} out of ${nTimes}. Here's the first error.\n${rest}`;
```

Comment:
Some reason to remove `chalk` from here?

Reply:
Did not make sense to keep it in as I'm not sure what to chalkify. If there is something that deserves to leap out, of course we should add it back.